src/game/objects/bridge.c

```diff
@@ -150,15 +181,25 @@ void BridgeTilt1Ceiling(
 void BridgeTilt2Floor(
     ITEM_INFO *item, int32_t x, int32_t y, int32_t z, int16_t *height)
 {
+    if (g_Config.fix_bridge_grabs && !IsSameBridgeSector(x, y, z, item)) {
+        return;
+    }
+
     int32_t level = item->pos.y + (GetOffset(item, x, z) >> 1);
-    if (y <= level) {
-        *height = level;
+    if (y > level) {
+        return;
     }
+
+    *height = level;
```

Comment:
I'm very afraid of this change, I feel like it has potential of having enormous impact and produce weird bugs.
Unless we specifically need it for this fix, I'd made a separate PR for it. WDYT?

Reply:
Ok. I made the change thinking it was related to the bug and wanted to match [Tomb5's version](https://github.com/Trxyebeep/TOMB5/blob/ac24665b4f7a4e1c321614d8a906c8fab5f49f17/TOMB5/game/objects.cpp#L215). I'm happy to remove the change because the bug was unrelated.

Though it is odd that the code mixes the types around. But you're right, if the change is needed, it should be a separate PR.

Reply:
Changed height back to int16_t and restored the use of constants and braces here.